unit/testing-utils/module_dependencies.txt

```diff
@@ -3,6 +3,7 @@ ansi-c
 catch
 goto-programs
 langapi
+libcprover-cpp
```

Comment:
I think the functionality related to testing `src/libcprover-cpp` should live in `unit/libcprover-cpp` not `unit/testing-utils`. This is particularly the case with the API, as it is desirable to be able to unit test the API without linking the API unit test binary to the rest of the code base on a `#include` level.